clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -495,30 +260,12 @@ void diagnostics()
   [[intel::private_copies(4, 8)]] unsigned int pc_two_arg[64];
 
   // numbanks
-  //CHECK: VarDecl{{.*}}numbanks
-  //CHECK: SYCLIntelMemoryAttr{{.*}}Implicit
-  //CHECK: SYCLIntelNumBanksAttr
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}8
-  //CHECK-NEXT: IntegerLiteral{{.*}}8{{$}}
-  [[intel::numbanks(8)]] unsigned int numbanks[32];
-
   //expected-error@+2{{attributes are not compatible}}
   [[intel::numbanks(16)]]
   [[intel::fpga_register]]
   //expected-note@-2 {{conflicting attribute is here}}
   unsigned int nb_reg[64];
 
-  //CHECK: VarDecl{{.*}}nb_nb 'unsigned int[64]'
-  //CHECK: SYCLIntelMemoryAttr{{.*}}Implicit Default
-  //CHECK: SYCLIntelNumBanksAttr
-  //CHECK-NEXT: ConstantExpr{{.*}}'int'
-  //CHECK-NEXT: value: Int 8
-  //CHECK-NEXT: IntegerLiteral{{.*}}'int' 8
-  //expected-warning@+2{{attribute 'numbanks' is already applied}}
```

Comment:
Do we test this elsewhere for numbanks?

Reply:
Yes, AST part we are checking inside clang/test/SemaSYCL/intel-fpga-local-ast.cpp file at line 223-231
Diagnostic part we are checking inside  clang/test/SemaSYCL/intel-fpga-local.cpp file at line 96-99